src/features/core/services/api.ts

```
@@ -1,38 +1,59 @@
+  },
+  async onResponse({ request, response, options }) {
+    if (response?.status === 401 && !options.retry) {
+      const refreshRes = await fetch(
+        'https://api.stg.idp.gistory.me/auth/refresh',
+        {
+          method: 'POST',
+          credentials: 'include',
+        },
+      ).catch(() => null);
+
+      if (refreshRes && refreshRes.ok) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)